osvr/RenderKit/osvr_display_configuration.h

```diff
@@ -132,6 +132,9 @@ class OSVRDisplayConfiguration {
     std::vector<float> const& getDistortionPolynomalBlue() const;
     ///@}
 
+    /// Returns the desired number of triangles in the constructed distortion mesh.
+    int OSVR_RENDERMANAGER_EXPORT getDesiredDistortionTriangleCount() const;
```

Comment:
I think the EXPORT define is actually supposed to come before the return type...